### PR TITLE
Translation of articles/README.md in french

### PR DESCRIPTION
**Pull request by almet**
Hi, I was browsing the docs and noticed that this page was left untranslated in french, so I did a quick translation. Hope it helps.

### CODE REVIEW

File: fr/user/articles/README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 Articles
 ========
-Articles are the core feature of wallabag. wallabag's content parser, named Graby gets the content from articles, and only the content.
+Les articles sont une fonctionalités clé dans wallabag. Graby, le *parser* de contenus de wallabag récupère les contenus des articles, et uniquement le contenu.
 
-## Mark articles as read
+## Marquer les articles comme « lu »
```
Comment (Kdecherf) [inline fr/user/articles/README.md:5]: s/lu/lus/

File: fr/user/articles/README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 Articles
 ========
-Articles are the core feature of wallabag. wallabag's content parser, named Graby gets the content from articles, and only the content.
+Les articles sont une fonctionalités clé dans wallabag. Graby, le *parser* de contenus de wallabag récupère les contenus des articles, et uniquement le contenu.
```
Comment (Kdecherf) [inline fr/user/articles/README.md:3]: I would prefer "représentent la caractéristique principale de". Also "parser" could be translated into its french counterpart "parseur" or "extractreur"

File: fr/user/articles/README.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 
-## Mark articles as read
+## Marquer les articles comme « lu »
 
-When you mark an article as read, it gets out of the category « Unread » and gets puts in the « Archive » category.
+Lorsque vous marquez un article comme « lu », il est déplacé depuis la catégorie « Non lus » vers la catégorie « Archive »
```
Comment (Kdecherf) [inline fr/user/articles/README.md:7]: The french translation of "Archive" in the menu is "lus", see https://hosted.weblate.org/translate/wallabag/messages/fr/?checksum=3d0e5c67d5bae135&q=archive&sort_by=-priority%2Cposition

**Comment by almet** [comment]
(updated)

File: fr/user/articles/README.md (hunk 2 above)
Comment (almet) [reply fr/user/articles/README.md]: J'ai mis « sont un concept clé de wallabag »

**Review by Kdecherf — APPROVED** [review]
:+1:

File: fr/user/articles/README.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
 Articles
 ========
-Articles are the core feature of wallabag. wallabag's content parser, named Graby gets the content from articles, and only the content.
+Les articles sont un concept clé de wallabag. Graby, l'extracteur de contenus de wallabag récupère les contenus des articles, et uniquement le contenu.
```
Comment (j0k3r) [inline fr/user/articles/README.md:3]:
```suggestion
Les articles sont un concept clé de wallabag. Graby, l'extracteur de contenus de wallabag, récupère le contenu des articles et uniquement le contenu.
```

**Comment by j0k3r** [comment]
Looks great, you can squash :+1:

**Review by Kdecherf — APPROVED** [review]

**Comment by Kdecherf** [comment]
Thanks @almet :)

**Comment by nicosomb** [comment]
Thank you @almet 😊

**Pull request merged**